test-requirements.txt

```diff
@@ -0,0 +1,7 @@
+flake8==2.4.1
+mccabe==0.3.1
+mock==1.3.0
+nose==1.3.7
```

Comment:
I think just using pytest should be enough -- is there something in Nose that we need?